### ISSUE TITLE
No release for version 9.3.0 and 9.3.1 in GitHub

### ISSUE THREAD

**Issue by GerardSmit** [dnnsoftware/Dnn.Platform#2663]
From what I can see in the [GitHub tags](https://github.com/dnnsoftware/Dnn.Platform/tags) the latest tag is `v9.3.1`, but in the [releases](https://github.com/dnnsoftware/Dnn.Platform/releases) there is only a release for the RC of version v9.3.1. This is also the case for v9.3.0.

Because of this, we don't get the new symbols, upgrade, install and deploy ZIP-files.  
Also the download link on the website (https://www.dnnsoftware.com/community/download) redirects to the lastest version, which is version v9.2.2 because both v9.3.0 and v9.3.1 are pre-releases.

Is there any reason why we don't get releases anymore for full releases and only tags?

**Comment by ohine** [comment]
We are currently in the process of getting v9.3.1 released as a production build, hopefully it will be released today. Recently we have changed the release process to include release candidates with a defined testing window for each release. The v9.3.0 release went through 8 weeks of RC test cycles, and unfortunately hours after it was released, a few issues related to an incomplete upgrade were reported. We made the decision to pull the v9.3.0 release, while quickly turning out a fix and getting a new RC out labeled v9.3.1.

If you have the ability to safely test the release candidates, it will help us deliver issues that might not be caught during our internal testing cycles. Each issue found, we update our processes to make the next build even better.
